java/core/src/java/org/apache/orc/impl/OutStream.java

```diff
@@ -399,4 +399,3 @@ public void suppress() {
     receiver.suppress();
   }
 }
-
```

Comment:
Ditto. Please revert this from this PR.

Reply:
yes I think it is already updated from the newer commit as leaving last line of a file blank is actually required by the checkstyle.